app/src/main/java/com/ciandt/dragonfly/example/features/realtime/RealTimeActivity.kt

```diff
@@ -32,6 +33,8 @@ class RealTimeActivity : FullScreenActivity(), RealTimeContract.View {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_real_time)
 
+        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
```

Comment:
What do you think about move this to FullScreenActivity? This can be a parameter `keepScreenOn = true`.